### ISSUE TITLE
Web UI

### ISSUE THREAD

**Issue by h2non** [vinxi/webui#1]

**Comment by h2non** [comment]
@zzarcon I'm sure you can help with this a lot. Think aboy a Cherokee like web admin dashboard provinding real time monitoring and configiration.

**Comment by h2non** [comment]
This will implicitely require that vinci exposes an API (maybe even REST) to third-party management.
Also, all the middleware layer should provide an effective way for instrumentation.

**Comment by h2non** [comment]
I was thinking that we should create here a serverless webapp using Angular, React or other frontend framework for fast UI iteration and features. Then the webapp will consume the HTTP API exposed via httpapi package.

@zzarcon It's mostly up to you how to develop this, but we can the details. I feel very confortable working with Angular, but I'm very open to use other solutions.

In any case, we have no hurry here since we have to design and code a lot of other packages first.

**Comment by zzarcon** [comment]
Yeah, I will do this for sure, but as you said, isn't too early for start with the webapp? Anyways I'd like to know more details about what the webapp is suposed to do.
